### PR TITLE
Use flash messages from AdminBundle

### PR DESCRIPTION
**Pull request by rn0**

### CODE REVIEW

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
i'm afraid that you forgot to add messages to translation files

**Comment by chives** [comment]
@JarekW they have always been there - https://github.com/fsi-open/admin-security-bundle/tree/master/Resources/translations

**Comment by JarekW** [comment]
:+1:

File: Behat/Context/Page/Element/FlashMessage.php [hunk 1]
```diff
@@ -6,5 +6,5 @@
 
 class FlashMessage extends Element
 {
-    protected $selector = array('css' => '.flash');
+    protected $selector = array('css' => '.alert');
```
Comment (szymach) [inline Behat/Context/Page/Element/FlashMessage.php]: notice: we don't even support PHP 5.3, so we can safely move to `[]` arrays

**Comment by szymach** [comment]
:+1:

**Pull request merged**